docs/modules/ROOT/pages/references/composition/dbaas/exoscale/common.adoc

```diff
@@ -0,0 +1,28 @@
+= pkg.appcat.composition.dbaas.exoscale
+
+The parent key for all of the following parameters is `pkg.appcat.composition.dbaas.exoscale`.
+
+Parameters under this parent key apply to all compositions of `pkg.appcat.composition.dbaas.exoscale.*` services.
+
```

Comment:
I'm not sure if the service is really called `dbaas`. Do you have a link or some other source?

Reply:
it's DBaaS Services on Exoscale.

Reply:
Yes I know Exoscale calls it DBaaS. But does _VSHN_ call it that too?
Compare with S3, VSHN calls the AppCat service "Object storage" : https://products.docs.vshn.ch/products/appcat/objectstorage.html

Reply:
@Kidswiss can you help here with the naming? Was it decided how to name the exoscale DBaaS Services in AppCat?

Reply:
Yes, we'll use DBaaS for this.

Reply:
I guess then it's ok as is?